bigbluebutton-html5/imports/api/users/server/handlers/validateAuthToken.js

```diff
@@ -45,7 +45,8 @@ export default function handleValidateAuthToken({ payload }) {
         addWelcomeChatMessage(meetingId, userId);
       }
 
-      return Logger.info(`Validated auth token as '${validStatus}' user=${userId} meeting=${meetingId}`);
+      return Logger.info(`Validated auth token as '${validStatus}'
+        user=${userId} meeting=${meetingId}`);
```

Comment:
This is how it gets rendered in the log

![sample1](https://cloud.githubusercontent.com/assets/22058534/26225503/669cdada-3bf5-11e7-9e97-3b6da03764c4.jpg)